### ISSUE TITLE
Adding selected multi-request support from History tab

### ISSUE THREAD

**Issue by dwisiswant0** [projectdiscovery/nuclei-burp-plugin#9]
This will alow to select multiple HTTP request from burp history and create single template with multiple request in it.

Selection example: 

<img width="639" alt="image" src="https://user-images.githubusercontent.com/8293321/155981784-5152acf3-5dd7-4264-b25b-3e6dd972fdb4.png">

Template example:

```yaml
id: template-id

info:
  name: Template Name
  author: geekboy
  severity: info
  description: description
  reference: https://
  tags: tags

requests:
  - raw:
      - |+
        POST /github/collect HTTP/2
        Host: {{Hostname}}
        Content-Length: 1000

        {"REDACTED"}

      - |+
        GET /projectdiscovery/nuclei/releases HTTP/2
        Host: {{Hostname}}

      - |+
        POST /_private/browser/stats HTTP/2
        Host: {{Hostname}}
        Content-Length: 2000

        {"REDACTED"}

    matchers-condition: and
    matchers:
      - type: word
        words:
          - "example"

      - type: status
        status:
          - 200
```

**Comment by forgedhallpass** [comment]
@dwisiswant0 this is now implemented and released under [v1.1.0](https://github.com/projectdiscovery/nuclei-burp-plugin/releases/tag/v1.1.0)

**Comment by dwisiswant0** [comment]
Kudos to y'all guys! :clap:
